app/migrations/seed.py

```diff
@@ -269,6 +304,11 @@ def create_ussd_menus():
         description='The token exchange amount is insufficient',
     )
 
+    update_or_create_menu(
+        name='exit_account_creation_prompt',
+        description='The self signup process has been initiated and the account is being created.'
```

Comment:
"you will get an sms when completed?"

Reply:
Yes. The user will get an SMS when the account is created.

Reply:
heh, good. My point was should that be part of the written feedback to the user.